STEER/Utilities/alihadd.sh

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env bash
+# source $ALICE_ROOT/libexec/alihadd.sh
+# Example usage:
+#   source $ALICE_ROOT/libexec/alihadd.sh
+#    * Filter input list before merging
+#       makeFilteredList input.list highPt filter.log
+#    *  Internal function
+#       testFileList input.list highPt | tee  filter.log
+
+
+loadUtilities(){
+    #
+    #  load utilities if not yet done
+    if [ -z `type -t alilog_info` ]; then
```

Comment:
Please comply with the bashism: double square braces and `$(...)`. The rest is for POSIX compliance but we do not care about that as we use bash, which is a better standard since 15 years.